lib/api/v3/work_packages/work_packages_by_project_api.rb

```diff
@@ -44,19 +44,50 @@ def create_service
                   project: @project,
                   send_notifications: !(params.has_key?(:notify) && params[:notify] == 'false'))
             end
+
+            def set_filters_from_json(query, json)
```

Comment:
Assignment Branch Condition size for set_filters_from_json is too high. [17.72/15]